pbft/src/vendor/manifest

```diff
@@ -1243,13 +1243,30 @@
 			"path": "protobuf/field_mask",
 			"notests": true
 		},
+		{
```

Comment:
hey in this same commit, could we address the "dependency strip down to minimal" issue?

Reply:
do you have any quick way to hack for gvt remove?